.github/workflows/main.yml

```diff
@@ -29,10 +29,14 @@ jobs:
         with:
           ocaml-compiler: ${{ matrix.ocaml-compiler }}
 
+      - run: opam install ocamlformat.0.27.0
```

Comment:
I think rather than explicitly add it here, we should add ocamlformat to the list of dependancies in the dune-project file. Whilst this works, if we modify the dune-project file it means people installing the dependancies manually will also get it.